- useEffect 콜백 함수는 프라미스를 리턴하면 안된다.

```jsx
const response = await fetch()

if(!response.ok) {
  throw new Error('Somthing went wrong')
}

// try catch로 감싸서 해결하지말고 Promise를 리턴하기 때문에 fetchMeals.catch()를 사용하는게 맞다.
```

